### PR TITLE
codeowners: add more people

### PR DESCRIPTION
**Pull request by mwasko**
More people to receive a notification when a change to any file is being made.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -2,4 +2,4 @@
 #At this time, the following people are notified and are expected to review the PRs:
 #Liam Girdwood (technical review), Anton Bobkov (technical/grammatical/style review), and Deb Taylor (grammatical/style review).
```
Comment (cujomalainey) [inline CODEOWNERS:3]: And their names here too

**Review by cujomalainey — CHANGES_REQUESTED** [review]

**Review by greg-intel — APPROVED** [review]

File: CODEOWNERS [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+# This file identifies people who are automatically notified when Pull Requests are made for /sof-docs. 
+# At this time, the following people are notified and are expected to review the PRs:
+# Liam Girdwood (technical review), Anton Bobkov (technical/grammatical/style review),
+# Deb Taylor (grammatical/style review), Marcin Maka (technical review)
+# and Michal Wasko (technical review).
 
-* @lgirdwood @anton-intel @deb-intel @intelkevinputnam @greg-intel
+# So if a pull request only touches javascript files, only these owners
+
+* @lgirdwood @anton-intel @deb-intel @intelkevinputnam @greg-intel @mmaka1 @mwasko
```
Comment (deb-intel) [inline CODEOWNERS:9]:
```suggestion
* @lgirdwood @deb-intel @intelkevinputnam @greg-intel @mmaka1 @mwasko
```

Comment (deb-intel) [reply CODEOWNERS:9]: Anton no longer works at Intel so please feel free to remove his name.

**Review by deb-intel — CHANGES_REQUESTED** [review]
Thanks for updating this.

File: CODEOWNERS (hunk 2 above)
Comment (mwasko) [reply CODEOWNERS]: @deb-intel I have updated the PR and removed Anton from the list.

File: CODEOWNERS (hunk 1 above)
Comment (mwasko) [reply CODEOWNERS]: @cujomalainey names added

**Review by cujomalainey — APPROVED** [review]

**Pull request merged**